server/helpers/mysql.ts

```diff
@@ -4,6 +4,8 @@ import Connection from 'mysql/lib/Connection';
 import bluebird from 'bluebird';
 import parse from 'connection-string';
 
+require('dotenv').config()
```

Comment:
I think you don't need to import this twice, the config vars should be already available from /index.ts isn't?